components/Carbon.js

```diff
@@ -99,6 +99,24 @@ class Carbon extends React.Component {
       viewportMargin: Infinity,
       lineWrapping: true
     }
+
+    let background = ''
+    let { codeMirrorNode } = this
+    if (codeMirrorNode) {
+      let [codeMirrorEditor] = codeMirrorNode.ref.children
+      let backgroundColor = getComputedStyle(codeMirrorEditor).backgroundColor
```

Comment:
Are you sure this is working? It looks like the background color's always being set to black, regardless of syntax theme.

Reply:
@jakedex, sorry for the late response.

I have made some changes and pushed. The problem with the earlier change was that when a theme was changed, the background calculated from the previous theme was still being applied.

I have addressed it in today's change but I am not able to reset the transparency slider once a new theme is selected. Any ideas?